open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -242,6 +242,11 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
is it still needed here? I only saw it used in the gain ranger plugin as a transitive dependency, so it would need to be in dependencyManagement but not anywhere else?

Reply:
Actually I remember now, this was the original place where I wanted to remove it because it looks like leftover from previous usage but then got too optimistic =). Should be good now only present in the master pom dependencies management.